File: src/main/java/com/devsuatt/todoApp/dto/TaskDto.java
package com.devsuatt.todoApp.dto;

import com.devsuatt.todoApp.model.TaskType;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.time.LocalDateTime;

@NoArgsConstructor
@AllArgsConstructor
@Getter @Setter
public class TaskDto {
    private String id;
    private String header;
    private String description;
    private LocalDateTime transactionDate;
    private TaskType taskType;
}

File: src/main/java/com/devsuatt/todoApp/dto/TaskDto.kt
package com.devsuatt.todoApp.dto

import com.devsuatt.todoApp.model.Category
import com.devsuatt.todoApp.model.TaskType
import com.devsuatt.todoApp.model.User
import java.time.LocalDateTime

data class TaskDto(

    val id: String?,
    val header: String?,
    val description: String?,
    val transactionDate: LocalDateTime?,
    val taskType: TaskType? = TaskType.TODO,
//    val category: Category?,

    )
